app/assets/stylesheets/custom/deck.css.scss

```diff
@@ -248,3 +248,15 @@ th .header{
 .archived{
   display: none;
 }
+#notes{
+  width: 60%;
+  margin: 20px auto 40px auto;
+}
+.create-archtype{
+  cursor: pointer;
+  color: #2980b9;
```

Comment:
Color literals like `#2980b9` should only be used in variable declarations; they should be referred to via variable everywhere else.